### ISSUE TITLE
can't connect to controller

### ISSUE THREAD

**Issue by hibeck** [motoz/nbetest#2]
Very nice script!
I'am using it to control/monitor my burner trough OpenHAB.

But I can't connect to the controller after updating to latest firmware 10.0929.
If I downgrade the controller to 10.0926 everything works fine again.

maybe something to do with changes in UDP-thing on the controller according to stokerforum.dk

**Comment by motoz** [comment]
Yes, firmware x.0x29 and up have a different protocol. There are some additional header fields and it optionally supports encrypting the header and data when changing settings to protect the password. I have a working version but that only works with the new protocol so I had not pushed it. And then I learned that there are still protocol changes coming soon, so it might not be worth it to support this version.

Anyway I pushed a separate branch that works for 7.0629/7.0929/10.0929 and up to at least x31, I don't know yet when the next protocol version is coming. https://github.com/motoz/nbetest/tree/fwversionx29. This firmware requires a correct password when changing settings (which previously was not checked at all).

Nice that you are using it with openHAB, I was hoping it would be useful for exactly such things!
